csv/generic/shared/src/main/scala-2/fs2/data/csv/generic/internal/MapShapedCsvRowDecoder.scala

```diff
@@ -74,6 +74,7 @@ private[generic] trait LowPriorityMapShapedCsvRowDecoder1 {
                           default: Option[Head] :: DefaultTail,
                           anno: Anno :: AnnoTail): DecoderResult[FieldType[Key, Head] :: Tail] = {
         val head = row(anno.head.fold(witness.value.name)(_.name)) match {
+          case Some(head) if head.isEmpty && default.head.nonEmpty => default.head.get.asRight
```

Comment:
I'd love to avoid the `.get` if possible. Something like:

```suggestion
          case Some(head) if head.isEmpty && default.head.nonEmpty => default.head.toRight(new DecoderError("Should not happen", row.line))
```